resident/resident-service/src/main/java/io/mosip/resident/util/Utilitiy.java

```diff
@@ -1,198 +1,202 @@
-                    ResidentErrorCode.RESIDENT_SYS_EXCEPTION.getErrorMessage(), e);
-        }
-        logger.debug(LoggerFileConstant.APPLICATIONID.toString(), LoggerFileConstant.UIN.name(), id,
-                "Utilitiy::getMailingAttributes()::exit");
-        return attributes;
-    }
-
-    public String getMappingJson() {
-        if (StringUtils.isEmpty(regProcessorIdentityJson)) {
-            return residentRestTemplate.getForObject(configServerFileStorageURL + getRegProcessorIdentityJson, String.class);
-        }
-        return regProcessorIdentityJson;
-    }
-
-}
+package io.mosip.resident.util;
+
+import io.mosip.kernel.core.exception.ServiceError;
+import io.mosip.kernel.core.http.ResponseWrapper;
+import io.mosip.kernel.core.logger.spi.Logger;
+import io.mosip.kernel.core.util.StringUtils;
+import io.mosip.resident.config.LoggerConfiguration;
+import io.mosip.resident.constant.ApiName;
+import io.mosip.resident.constant.LoggerFileConstant;
+import io.mosip.resident.constant.ResidentErrorCode;
+import io.mosip.resident.dto.IdRepoResponseDto;
+import io.mosip.resident.dto.JsonValue;
+import io.mosip.resident.exception.ApisResourceAccessException;
+import io.mosip.resident.exception.IdRepoAppException;
+import io.mosip.resident.exception.ResidentServiceCheckedException;
+import io.mosip.resident.exception.ResidentServiceException;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * @author Girish Yarru
+ * @version 1.0
+ */
+
+@Component
+public class Utilitiy {
+
+    private static final Logger logger = LoggerConfiguration.logConfig(Utilitiy.class);
+
+    @Autowired
+    private ResidentServiceRestClient residentServiceRestClient;
+
+    @Autowired
+    private TokenGenerator tokenGenerator;
+
+    @Value("${config.server.file.storage.uri}")
+    private String configServerFileStorageURL;
+
+    @Value("${resident.identityjson}")
+    private String getRegProcessorIdentityJson;
+
+    @Value("${mosip.primary-language}")
+    private String primaryLang;
+
+    @Value("${mosip.secondary-language}")
+    private String secondaryLang;
+
+    @Value("${mosip.notification.language-type}")
+    private String languageType;
+
+    @Autowired
+    private RestTemplate residentRestTemplate;
+
+    @Autowired
+    private Environment env;
+
+    private static final String IDENTITY = "identity";
+    private static final String VALUE = "value";
+    private static String regProcessorIdentityJson = "";

```

Comment:
If util is changed, why is this changed ?

Reply:
It was changed due to formatting. The only line change is     private static String regProcessorIdentityJson = "";